### DISCUSSION TITLE
Issues in a repo don't show the author, it shows the repo instead

### DISCUSSION THREAD

**Discussion by DarkIntaqt** [community/community#29366] [Mobile]
Is it intended that the issue tracker on mobile only shows the repo again, like I forgot where I clicked a single step earlier on?
![img](https://user-images.githubusercontent.com/61588850/184508019-19b78a8d-24c0-4f77-8592-b3a5e0160ffa.jpeg)
This image shows only the repo, I think this is misleading, at least for me. I would rather display something like this:
![idea](https://user-images.githubusercontent.com/61588850/184508103-de866b11-6070-4206-9309-b133d4eaf55d.jpeg)

What is your opinion about it? Happened on iOS. Hope this is not a duplicate, can't find anything related to the keywords "repo author mobile".

**Comment by haldun** [comment]
Hi @DarkIntaqt, thanks for your feedback. I will pass this onto to the team for evaluation.
